src/lib/Microsoft.Health.Fhir.Ingest/Exceptions/CompressionNotSupportedException.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Common.Telemetry;
+using Microsoft.Health.Common.Telemetry.Exceptions;
+
+namespace Microsoft.Health.Fhir.Ingest.Exceptions
+{
+    public class CompressionNotSupportedException : IomtTelemetryFormattableException
```

Comment:
Does this mean it is customer facing?  If so I think that is incorrect since this wouldn't be actionable to the customer

Reply:
No it won't be customer facing since IomtTelemetryFormattableException is not customer facing by default. If we were to instead inherit from [ThirdPartyLoggedFormattableException ](https://github.com/microsoft/iomt-fhir/blob/0ecfa7336df1516844bce009725f769f37b35510/src/lib/Microsoft.Health.Common/Telemetry/Exceptions/ThirdPartyLoggedFormattableException.cs) which inherits from IomtTelemetryFormattableException 
 then it would be customer facing.

Reply:
confirmed it does not get logged as a metric either. not sure we really need to inherit from IomtTelemetryFormattableException, though doing so does give some context to the developer about the severity, error type, stage even though it is not surfaced to an end user

Reply:
Thanks for confirming!  Sorry I was misremembering the content from Artun's prior PR.